queries.sql

```diff
@@ -25,3 +25,76 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 /* QUERY TABLE FOR ANIMALS WITH WEIGHT BETWEEN 10.4KG AND 17.3KG */
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+
+/** DAY 2 ***/
+
+/*** TRANSACTIONS ***/
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+select * from animals;
+COMMIT;
+select * from animals;
+
+-- update other species to pokemon
+BEGIN;
+UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+UPDATE 5
+select * from animals;
+COMMIT;
+select * from animals;
+
+
+-- delete all animals
+BEGIN;
+DELETE FROM animals;
+DELETE 11
+select * from animals;
+ROllBACK;
+
+--delete animals born after 2022 and update weight_kg
+
+begin;
+BEGIN
+delete from animals where date_of_birth > DATE '2022-01-01';
+DELETE 1
+
+--savepoint
+SAVEPOINT sp1;
+SAVEPOINT
+
+--update all weight_kg
+update animals set weight_kg = weight_kg * -1;
+UPDATE 10
```

Comment:
- [ ] What does this `UPDATE 10` code do? I believe it is unnecessary. I would really like to learn about what it does. Do let me know in this comment. If not you may want to remove it.